### ISSUE TITLE
Add Framework support for GKE

### ISSUE THREAD

**Issue by AdonisOkpidi** [kubescape/helm-charts#567]
## Overview
Our project requires installing Kubescape in both GKE and AKS. However, there is currently no dedicated GKE framework available in Kubescape.

## Problem
To meet the project requirements, I am planning to create a privileged pod to use alongside the `CIS v1.23` and `AllControls` Kubescape frameworks (When this is supported in [feature request](https://github.com/kubescape/helm-charts/issues/566)) While this approach works, it requires manually filtering reports to exclude controls related to managed services. Introducing a specific framework for GKE would significantly reduce this effort, as most necessary controls already exist, with the exception of those related to Managed Services compliance.

## Solution
Develop a GKE-specific framework similar to the `cis-aks-t1.2.0` framework for AKS. This tailored framework would simplify compliance processes, eliminate the need for manual filtering, and ensure a smoother integration of GKE-specific controls.

**Comment by slashben** [comment]
Hey @AdonisOkpidi , to clarify: are you looking for CIS GKE framework (like the one mentioned [here](https://cloud.google.com/kubernetes-engine/docs/concepts/cis-benchmarks) or [here](https://cloud.google.com/kubernetes-engine/docs/concepts/cis-benchmarks)?

**Comment by AdonisOkpidi** [comment]
@slashben Yes correct! You can find the details on the official website [CIS workbench](https://workbench.cisecurity.org/dashboard). The current version is `v1.7.0`

![CIS_screenshot0](https://github.com/user-attachments/assets/6226ca86-ac93-42e8-92c3-9191821128e2)
![CIS_screenshot](https://github.com/user-attachments/assets/c2731692-679c-4756-8ce4-7a26dc8a94c4)
![CIS_screenshot2](https://github.com/user-attachments/assets/e55deeb9-fb7a-434c-bd43-e759cdf7614f)
![CIS_screenshot3](https://github.com/user-attachments/assets/698e846e-0b86-42d7-81df-c0ce782393ce)

**Comment by AdonisOkpidi** [comment]
@slashben Happy new year. 

I am just checking in to see the timeline of this?

**Comment by slashben** [comment]
Happy New Year to you too!

We are waiting for a contributor on this therefore regreably I don't have
an ETA :-(
